docs/supported-libraries.md

```diff
@@ -28,7 +28,7 @@ These are the supported libraries and frameworks:
 | [Apache CXF JAX-RS Client](https://cxf.apache.org/)                                                                               | 3.0+                           |
 | [Apache CXF JAX-WS](https://cxf.apache.org/)                                                                                      | 3.0+                           |
 | [Apache Dubbo](https://github.com/apache/dubbo/)                                                                                  | 2.7+ (not including 3.x yet)   |
-| [Apache HttpAsyncClient](https://hc.apache.org/index.html)                                                                        | 4.1+                           |
+| [Apache HttpAsyncClient](https://hc.apache.org/index.html)                                                                        | 4.1+ and 5.0+                  |
```

Comment:
I think remove this change (5.0+ is covered by 4.1+ anyways)

Reply:
done